src/lib/services/account/lcd.ts

```diff
@@ -18,3 +21,19 @@ export const getAccountDataLcd = async (
     icns,
   };
 };
+
+export const getAccountTypeLcd = async (
+  endpoint: string,
+  address: BechAddr
+) => {
+  const accountTypeLcd = await axios
+    .get(`${endpoint}/cosmos/auth/v1beta1/accounts/${address}`)
+    .then((res) => parseWithError(zAccountTypeLcd, res.data));
```

Comment:
To make it more consistency with others part of the application. Please consider extract `res`
`.then(({ data }) => parseWithError(zAccountTypeLcd, data));`